agent/discovery/host.go

```diff
@@ -90,3 +89,35 @@ func getHostIpAddresses() ([]string, error) {
 
 	return ipAddrList, nil
 }
+
+func GetOSVersion() string {
+	infoStat, err := host.Info()
+	if err != nil {
+		log.Errorf("Error while getting host info: %s", err)
+	}
+	return infoStat.PlatformVersion
+}
+
+func getTotalMemoryMB() int {
```

Comment:
No particular reason. No one is using it, except for this discovery.

When neede It could very well be public, or even an Interface to a service.
I kept it really simple.

Is that good enough for now?

Reply:
I was just surprised because the other `Get...` function are using capital letter. I just noticed the difference and lack of consistency. Nothing else hehe

Reply:
Ah, my bad, now I spotted.

They were supposed to be all private actually. Now I see I have all public except for one.
A leftover. Gonna make all private.

Thanks :pray: